pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -104,15 +110,87 @@ def init(self):
     return
 
 
-def list_files(tag=None, sat_id=None, data_path=None, format_str=None):
+def clean(self):
+    """Return COSMIC GPS data cleaned to the specified level.
+
+    Parameters
+    ----------
+    self : pysat.Instrument
+        Instrument class object, whose attribute clean_level is used to return
+        the desired level of data selectivity.
+
+    Note
+    ----
+    Supports 'clean', 'dusty', 'dirty'
+
+    """
+    if self.tag == 'ionprf':
+        # ionosphere density profiles
```

Comment:
```suggestion
        # Ionosphere density profiles
```

Reply:
These comment suggestions are good but it is an unfortunate bit of timing that many of these were changed in the active pulls and just aren't here yet. Currently trying to reproduce a bug Jeff is having with file dates in the xarray loading pull #18. Since that pull doesn't change the list_files code it is likely also here.